src/help/h_jsonsettings.php

```diff
@@ -117,11 +114,9 @@ class=\"settings-jpath\">decision</code> is an array with one entry per
 }</code></pre></li>
 
 
-<li><p><strong>Copying settings between conferences.</strong> Beware of IDs
-when copying settings between conferences. Unless you are careful, the IDs
-used in one conference may differ from the IDs in another. Consider removing
-the IDs from a conference’s JSON settings before uploading those settings to
-another conference.</p></li>
+<li><p><strong>Copying settings between programs.</strong> Beware of IDs
```

Comment:
Could you not change the line breaks but just the text? again easier for future merge.